pkg/nanny/timer.go

```diff
@@ -73,7 +90,7 @@ func (nt *Timer) onExpire() {
 	nt.lock.Unlock()
 }
 
-func (nt *Timer) notify() error {
+func (nt *Timer) notify(isAllClear bool) error {
```

Comment:
I would prefer new function `notifyAllClear` rather, or even new Timer type for the allClear message.